### DISCUSSION TITLE
Simple peg-in-hole model

### DISCUSSION THREAD

**Discussion by JeanElsner** [google-deepmind/mujoco#879] [Show and tell]
I've seen some discussion surrounding this topic and therefore I'd like to share with you this simple peg-in-hole [model](https://github.com/deepmind/mujoco/files/11530245/peg-in-hole.zip) we created. I tried using V-HACD to create the convex decomposition but couldn't get it to work (the peg wouldn't fit), therefore I created the collision meshes by hand (Blender file included). We also printed the pieces (they fit into the Panda grippers) and evaluated the model with hardware-in-the-loop. Maybe some of you can figure out how to produce the decomposition automatically, the manual modeling is a bit of a hassle (and this is a very simple model).

[Video](https://www.youtube.com/watch?v=PHuBPm6bNWU)
[Video with robot hardware](https://www.youtube.com/watch?v=ek-oO5oX3Uo)

**Comment by yuvaltassa** [comment]
Very nice!

As mentioned in a different issue, [this](https://github.com/SarahWeiii/CoACD) is a fairly new package that seems to be significantly better than V-HACD.

**Comment by kevinzakka** [comment]
cc @carlosferrazza

**Comment by hai-h-nguyen** [comment]
Hi @JeanElsner, thanks for your model. It looks nice. Could you give me some instructions for doing that w/ a round peg in Blender? Thank you.

**Comment by NadeenElh** [comment]
Hello, @JeanElsner. Is the code for the model available anywhere?
